deploy/1-deploy.ts

```diff
@@ -3,7 +3,19 @@ import testnetDeployments from "@venusprotocol/venus-protocol/networks/testnet.j
 import hre, { ethers } from "hardhat";
 import { HardhatRuntimeEnvironment } from "hardhat/types";
 
-const ADDRESSES: any = {
+interface Network {
+  vBNBAddress: string;
+  comptroller: string;
+  WBNBAddress: string;
+  timelock: string;
+  acm: string;
+}
+
+interface Addresses {
+  [key: string]: Network;
+}
+
+const ADDRESSES: Addresses = {
```

Comment:
Its better to use `getContract` than hardcode addresses. This makes the deployment script more resilient because we don't have to extend this config when deploying to new networks and it allows the script to be tested and run with hardhat

Reply:
Also lets change the exports in this file to use module exports

Reply:
The first comment will also solve the outdated imports on line 1 and 2. 
We should update `@venusprotocol/venus-protocol` all the same

Reply:
We can get it by adding the governance package to external deployments

Reply:
We can this object completely